NEWS.txt

```diff
@@ -187,6 +187,9 @@ New features
     - Added snitch for Microsoft Azure of name AzureSnitch (CASSANDRA-18646)
     - legacy command line options from cassandra-stress were removed
     - `-mode` option in cassandra-stress has `native` and `cql3` as defaults and they do not need to be specified
+    - Allow to write the commitlog using direct I/O. Direct I/O is a new feature that minimizes cache effects by using
```

Comment:
I would add "and memory-mapping overhead" after "cache effects" as this is a much bigger problem with the mmap option.